src/components/QueryBuilder.vue

```diff
@@ -75,6 +75,20 @@ export default Vue.extend( {
 </script>
 
 <style scoped lang="scss">
+<!-- TODO replace with link component once available-->
```

Comment:
Needs to be an CSS comment, not an HTML comment. I failed to spot that earlier

```suggestion
// TODO replace with link component once available
```

Reply:
agh! of course. sorry about that!